Controller/Adminhtml/Cart/Data.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ *
+ * Copyright © 2013-2017 Bolt, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Bolt\Boltpay\Controller\Adminhtml\Cart;
+
+use Bolt\Boltpay\Helper\Cart as CartHelper;
+use Exception;
+use Magento\Framework\App\Action\Action;
+use Magento\Framework\App\Action\Context;
+use Magento\Framework\Controller\Result\JsonFactory;
+use Magento\Framework\Controller\Result\Json;
+use Magento\Framework\DataObjectFactory;
+use Bolt\Boltpay\Helper\Config as ConfigHelper;
+use Bolt\Boltpay\Helper\Bugsnag;
+
+/**
+ * Class Data.
+ * Create Bolt order controller.
+ *
+ * Called from the replace.phtml javascript block on checklout button click.
+ *
+ * @package Bolt\Boltpay\Controller\Cart
+ */
+class Data extends Action
+{
+    /**
+     * @var JsonFactory
+     */
+    private $resultJsonFactory;
+
+    /**
+     * @var CartHelper
+     */
+    private $cartHelper;
+
+    /**
+     * @var ConfigHelper
+     */
+    private $configHelper;
+
+    /**
+     * @var Bugsnag
+     */
+    private $bugsnag;
+
+    /**
+     * @var DataObjectFactory
+     */
+    private $dataObjectFactory;
+
+    /**
+     * @param Context $context
+     * @param JsonFactory $resultJsonFactory
+     * @param CartHelper $cartHelper
+     * @param ConfigHelper $configHelper
+     * @param Bugsnag $bugsnag
+     * @param DataObjectFactory $dataObjectFactory
+     *
+     * @codeCoverageIgnore
+     */
+    public function __construct(
+        Context $context,
+        JsonFactory $resultJsonFactory,
+        CartHelper $cartHelper,
+        ConfigHelper $configHelper,
+        Bugsnag $bugsnag,
+        DataObjectFactory $dataObjectFactory
+    ) {
+        parent::__construct($context);
+        $this->resultJsonFactory = $resultJsonFactory;
+        $this->cartHelper        = $cartHelper;
+        $this->configHelper      = $configHelper;
+        $this->bugsnag           = $bugsnag;
+        $this->dataObjectFactory = $dataObjectFactory;
+    }
+
+    /**
+     * Get cart data for bolt pay ajax
+     *
+     * @return Json
+     * @throws Exception
+     */
+    public function execute()
+    {
+        try {
+            // additional data collected from the (one page checkout) page,
+            // i.e. billing address to be saved with the order
+            $place_order_payload = $this->getRequest()->getParam('place_order_payload');
+            // call the Bolt API
+            $boltpayOrder = $this->cartHelper->getBoltpayOrder(true, $place_order_payload);
+
+            // format and send the response
+            $cart = [
+                'orderToken'  => $boltpayOrder ? $boltpayOrder->getResponse()->token : '',
+                'authcapture' => $this->configHelper->getAutomaticCaptureMode()
```

Comment:
can you hard code true here? In case of backend processing I think it'll be auto-captured.